hack/update-codegen-dockerized.sh

```diff
@@ -36,13 +36,7 @@ ANTREA_PKG="antrea.io/antrea"
 # https://github.com/kubernetes-sigs/gateway-api/issues/11
 ANTREA_PROTO_PKG="antrea_io.antrea"
 
-if [[ "$#" -eq 1 && $1 == "mockgen" ]]; then
-  mockgen
-  reset_year_change
-  exit 0
-fi
-
-function mockgen {
+function mockgencodes {
```

Comment:
maybe `generate_mocks` would be a better name, and consistent with `reset_year_change`?

Reply:
done